grails-app/i18n/messages.properties

```diff
@@ -4642,3 +4642,15 @@ react.cycleCount.filter.productCatalogs.label=Product Catalogue
 react.cycleCount.filter.abcClass.label=ABC Class
 react.cycleCount.filter.negativeInventory.label=Show negative quantity only
 react.cycleCount.filter.search.label=Search...
+
+
+# InboundV2
+react.inboundV2.validation.requestedBy.required.label=Requested by is required
```

Comment:
Can we just use existing inbound labels? IIRC theoretically we shouldn't have to change anything on the backend for this refactor, or the changes could be just minimal. Please refrain from using `V2` on a backend at all for now.